packages/workbox-precaching/src/PrecacheController.ts

```diff
@@ -40,6 +40,7 @@ class PrecacheController {
   constructor(cacheName?: string) {
     this._cacheName = cacheNames.getPrecacheName(cacheName);
     this._urlsToCacheKeys = new Map();
+    this._urlsToIntegrity = new Map();
```

Comment:
Why not just have a mapping between URLs and an object with `{cacheKey, integrity}` properties?

Reply:
Yeah, I got the naming wrong. This is effectively a `cacheKey` => `integrity` mapping, not a `url` => `integrity` mapping. (At least according to the distinction between `url` and `cacheKey` that's used elsewhere.)

I'll rename, and hopefully that explains why we can't just use a single `Map` with tuple values.